### PR TITLE
align content for grid-row/grid-column.

### PR DESCRIPTION
**Pull request by zhangenming**
### Description
https://github.com/mdn/content/issues/21199#issuecomment-1272189973

### Motivation
Keep documents consistent.

### CODE REVIEW

**Comment by zhangenming** [comment]
image

File: files/en-us/glossary/Grid_row/index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Grid Row
-slug: Glossary/Grid_Rows
+slug: Glossary/Grid_Row
```
Comment (teoli2003) [inline files/en-us/glossary/Grid_row/index.md:3]:
```suggestion
slug: Glossary/Grid_row
```

**Review by teoli2003 — APPROVED** [review]

**Review by teoli2003 — DISMISSED** [review]
You shouldn't change the slug manually. Please use yarn content move so that a redirect is added.

**Comment by zhangenming** [comment]
> You shouldn't change the slug manually. Please use yarn content move so that a redirect is added.

I'm sorry. I don't understand. What am I supposed to do

**Comment by bsmth** [comment]
> I'm sorry. I don't understand. What am I supposed to do

Hi @zhangenming there's some information here for moving documents: https://github.com/mdn/content#moving-one-or-more-documents

You need to run:

```bash
yarn content move <from-slug> <to-slug>
```

**Comment by zhangenming** [comment]
done? like this?

**Comment by bsmth** [comment]
> done? like this?

taking a look 👀

**Review by bsmth — APPROVED** [review]
Thanks @zhangenming 👍🏻

**Comment by bsmth** [comment]
@teoli2003 this one looks good to me if you want to double-check 👍🏻

**Comment by bsmth** [comment]
>bsmth removed the request for review from teoli2003
>bsmth requested a review from teoli2003

Sorry, the re-request review button was disabled

**Comment by bsmth** [comment]
Going to merge this one 👍🏻

**Pull request merged**